.github/workflows/vmware-releases-main.yml

```diff
@@ -0,0 +1,47 @@
+name: Get the latest open-vm-tools release for main
+on:
+  schedule:
+    - cron:  '0 7 * * 5'
+
+jobs:
+  get-vmware-release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Fetch latest open-vm-tools release
+        id: fetch-latest-release
+        run: |
+          git clone https://github.com/vmware/open-vm-tools
+          versionMain=$(git -C open-vm-tools ls-remote --tags origin | cut -f2 | sed -n "/refs\/tags\/stable-[0-9]*\.[0-9]*\.[0-9]*$/s/^refs\/tags\/stable-//p" | sort -ruV | head -n1)
+          buildNumber=$(curl -sSL https://api.github.com/repos/vmware/open-vm-tools/releases/latest | jq -r '.assets[0].name' | sed -n "s/^open-vm-tools-${versionMain}*-\([0-9]*\)\..*/\1/p")
+          rm -rf open-vm-tools
+          echo ::set-output name=BASE_BRANCH_MAIN::main
+          echo ::set-output name=BUILD_NUMBER::$(echo ${buildNumber})
+          echo ::set-output name=VERSION_MAIN::$(echo ${versionMain})
+      - name: Set up Flatcar SDK
+        id: setup-flatcar-sdk
+        run: .github/workflows/setup-flatcar-sdk.sh
+      - name: Apply patch for main
+        id: apply-patch-main
+        env:
+          TARGET: main
+          BASE_BRANCH: ${{ steps.fetch-latest-release.outputs.BASE_BRANCH_MAIN }}
+          BUILD_NUMBER: ${{ steps.fetch-latest-release.outputs.BUILD_NUMBER }}
+          PATH: ${{ steps.setup-flatcar-sdk.outputs.path }}
+          VERSION_NEW: ${{ steps.fetch-latest-release.outputs.VERSION_MAIN }}
+        run: .github/workflows/vmware-apply-patch.sh
+      - name: Create pull request for main
+        uses: peter-evans/create-pull-request@v3
+        if: steps.apply-patch-main.outputs.UPDATE_NEEDED == 1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          base: ${{ steps.fetch-latest-release.outputs.BASE_BRANCH_MAIN }}
+          branch: vmware-${{ steps.fetch-latest-release.outputs.VERSION_MAIN }}-main
```

Comment:
This `main` should be `${{ steps.fetch-latest-release.outputs.BASE_BRANCH_MAIN }}`

Reply:
Right, done